src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -345,6 +346,10 @@ private void OnHtmlSourceLoaded(object sender, EventArgs e)
 			Duration = NativeMethods.GetDuration(_activeElement.HtmlId);
 		}
 		OnSourceLoaded?.Invoke(this, EventArgs.Empty);
+
+		IsPause = NativeMethods.GetPaused(_activeElement.HtmlId);
```

Comment:
In line 344 you have a null check. But then here, you dereference `_activeElement` directly. Either the null check condition is a dead code, or this line may incorrectly throw if null is dereferenced.

Reply:
Done. null check add.